### ISSUE TITLE
Make scraping sequential, not parallel

### ISSUE THREAD

**Issue by philipmjohnson** [radgrad/radgrad2#764]
Please do work for this task in a branch called issue-764.

@ruizaj found during summer 21 that a cause of scraper fragility appears to be running them in parallel using async/await. There may be some kind of bad interaction among the runtimes. 

There is a not a big time savings to running things in parallel, so it is useful to explore if a sequential solution reduces the fragility of the scrapers.

**Comment by philipmjohnson** [comment]
duplicate of #749
